aps/rms_jobs/import_fields_from_disk.py

```diff
@@ -111,6 +111,9 @@ def field_name_from_full_name(full_field_name, zone_name, region_name=''):
 
 def load_field_values(field_name: str, path: Path, grid=None, debug_level=Debug.OFF):
     if path.suffix.upper() == '.GRDECL':
+        print('Deprecation warning:')
+        print("The file format 'grdecl' for field parameters is deprecated")
+        print("Please use 'roff' format instead.")
```

Comment:
Consider using the built-in deprecation warning

```suggestion
                                    warnings.warn(
                                        "The file format using 'grdecl' for field parameters is deprecated.\nPlease use 'roff' format.",
                                        DeprecationWarning,
                                    )
```